LCLSGeneral/LCLSGeneral/POUs/Logger/FB_LogHandler.TcPOU

```diff
@@ -97,9 +98,15 @@ END_IF
 IF NOT bAdapterSet THEN
     fbGetAdapterIP();
     IF NOT (fbGetAdapterIP.bBusy or fbGetAdapterIP.bError) THEN
-        fbSocket.sLocalHost := fbGetAdapterIP.arrAdapters[1].sIpAddr;
-        SocketEnable := TRUE;
-        bAdapterSet := TRUE;
+        FOR idxPortFind := 0 TO MAX_LOCAL_ADAPTERS DO
+            IF FIND(fbGetAdapterIP.arrAdapters[idxPortFind].sIpAddr,
+               GVL_Logger.sIpTidbit) <> 0 THEN
+                fbSocket.sLocalHost := fbGetAdapterIP.arrAdapters[idxPortFind].sIpAddr;
+                SocketEnable := TRUE;
+                bAdapterSet := TRUE;
+                EXIT;
+            END_IF
+		END_FOR
```

Comment:
I spy mixed tabs and spaces

Reply:
This repo is littered with this issue... Should address this elsewhere.